<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AbntFrame</class>
 <widget class="QWidget" name="AbntFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="connButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Iniciar</string>
   </property>
  </widget>
  <widget class="QScrollBar" name="activeBar">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>80</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>5</number>
   </property>
   <property name="pageStep">
    <number>5</number>
   </property>
   <property name="sliderPosition">
    <number>1</number>
   </property>
   <property name="tracking">
    <bool>true</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Energia Ativa</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Energia Reativa</string>
   </property>
  </widget>
  <widget class="QScrollBar" name="reactiveBar">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>110</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="maximum">
    <number>5</number>
   </property>
   <property name="pageStep">
    <number>5</number>
   </property>
   <property name="sliderPosition">
    <number>1</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>90</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>+ Pulso/seg</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
